src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.js

```diff
@@ -0,0 +1,260 @@
+const flextStartSyle = { justifyContent: 'flex-start' };
+const displaFlexAndjustifyContentSpaceBetweenStyle = {
+  display: 'flex',
+  justifyContent: 'space-between',
+};
+
+const upperCaseStyle = {
+  textTransform: 'uppercase',
+};
+
+const fieldNames = {
+  firstname: 'firstname',
+  lastname: 'lastname',
+  address: 'address',
+  city: 'city',
+  province: 'province',
+  country: 'country',
+  zipCode: 'zipCode',
+  phone: 'phone',
+  company: 'company',
+  industry: 'industry',
+};
+
+export const ContactInformation = InjectAppServices(
+  ({ dependencies: { dopplerUserApiClient }, handleSaveAndContinue, onComplete }) => {
+    const [state, setState] = useState({ loading: true });
+    const [formSubmitted, setFormSubmitted] = useState(false);
+    const createTimeout = useTimeout();
+    const intl = useIntl();
+
+    useEffect(() => {
+      const fetchData = async () => {
+        const contactInformationResult = await dopplerUserApiClient.getContactInformationData();
+
+        if (contactInformationResult.success) {
+          onComplete(contactInformationResult.value.completed);
+        }
+
+        setState({
+          contactInformation: contactInformationResult.success
+            ? contactInformationResult.value
+            : null,
+          success: contactInformationResult.success,
+          loading: false,
+        });
+      };
+      fetchData();
+    }, [dopplerUserApiClient]);
+
+    const _ = (id, values) => intl.formatMessage({ id: id }, values);
+    const defaultOption = { key: '', value: _('checkoutProcessForm.empty_option_select') };
+    const language = intl.locale;
+
+    const getFormInitialValues = () => {
+      let initialValues = Object.keys(fieldNames).reduce(
+        (accumulator, currentValue) => ({
+          ...accumulator,
+          [currentValue]: '',
+        }),
+        {},
+      );
+
+      if (state.contactInformation) {
+        initialValues = { ...state.contactInformation };
+      }
+
+      return initialValues;
+    };
+
+    const submitContactInformationForm = async (values, { setSubmitting }) => {
+      setFormSubmitted(false);
+
+      const result = await dopplerUserApiClient.createOrUpdateContactInformation(values);
+      if (result.success) {
+        setFormSubmitted(true);
+        createTimeout(() => {
+          setFormSubmitted(false);
+          handleSaveAndContinue();
+        }, 3000);
+      }
+    };
+
+    return (
+      <>
+        {state.loading ? (
+          <Loading />
+        ) : (
+          <div className="dp-wrapper-form-plans contact-information" style={containerStyle}>
+            <div style={displaFlexAndjustifyContentSpaceBetweenStyle}>
+              <h3 style={upperCaseStyle}>{_('checkoutProcessForm.contact_information_title')}</h3>
+            </div>
+            <Formik onSubmit={submitContactInformationForm} initialValues={getFormInitialValues()}>
+              <Form>
+                <fieldset>
```

Comment:
Maybe later they add the legend tag to each form? (accessibility)

Reply:
Done